### PR TITLE
docs: update project-lifecycle.md for grammar

### PR DESCRIPTION
**Pull request by barriebyron**
Hi, Hyperledger community! 
I made a few grammatical edits as I try out the OSS contributions process to your project.

I always enjoy lifecycle processes and think these changes tighten up your message and process while adhering to grammar rules and best practices.

Other technical writers are looking for ways to grow their blockchain skills, so I am pulling together a list of projects they might want to contribute to.

### CODE REVIEW

File: project-lifecycle.md [hunk 1]
```diff
@@ -7,24 +7,24 @@ nav_order: 6
...
 as a requirements or use cases document, a whitepaper, or analysis.
-Others may be to develop a new capability, or refactor (or remove) an
+Other projects develop a new capability, refactor, or remove an
 existing capability for the Hyperledger technology releases. Such
-projects may take the form of a new component (e.g. a new repository) or
-may propose additions, deletions or changes to an existing
-repository(s).
+projects may take the form of a new component (for example, a new repository) or
+may propose additions, deletions, or changes to an existing
+repository or repositories.
```
Comment (barriebyron) [inline project-lifecycle.md:18]: suggest we adhere to conventions as  https://developers.google.com/style/plurals-parentheses?hl=en

File: project-lifecycle.md [hunk 2]
```diff
@@ -7,24 +7,24 @@ nav_order: 6
 
 # Project Lifecycle
 
-The term "project" within Hyperledger will refer to a
+The term "project" within Hyperledger refers to a
```
Comment (barriebyron) [inline project-lifecycle.md:10]: Informing readers about project lifecycles require some verbs to use future tense. I suggest using present tense when possible, following https://developers.google.com/style/tense?hl=en

File: project-lifecycle.md [hunk 3]
```diff
@@ -43,40 +43,40 @@ and may go through several iterations.
...
+to the relevant project maintainers to handle as they deem fit.
 
 A Proposal must:
 
--   have a clear description
--   have a well-defined scope
--   identify committed development resources
--   identify initial maintainers
--   be vendor neutral
+-   Have a clear description
```
Comment (barriebyron) [inline project-lifecycle.md:54]: suggest capitalizing the first letter in a list item, following https://developers.google.com/style/lists?hl=en#numbered-lettered-bulleted-lists

File: project-lifecycle.md [hunk 4]
```diff
@@ -43,40 +43,40 @@ and may go through several iterations.
...
 
 # Incubation
 
 Approved project proposals enter into *Incubation*. For new
-components/modules, a repository will be created under the
+components and modules, a repository is created under the
 [Hyperledger Github org](https://github.com/hyperledger)
 and optionally in [JIRA](https://jira.hyperledger.org), if requested. New
-features/capabilities should be handled through pull requests labeled
+features/capabilities must be handled through pull requests labeled
```
Comment (barriebyron) [inline project-lifecycle.md]: should is ambiguous

**Review by barriebyron — COMMENTED** [review]
adding links to style guidance for the suggested changes, thanks for taking a look

**Comment by tkuhrt** [comment]
Hi, @barriebyron : Thank you for submitting the pull request. Can you please follow the steps listed under [checks](https://github.com/hyperledger/tsc/pull/28/checks?check_run_id=5664439725) to correct the DCO sign-off requirement. In the meantime, I will take a look and provide some thoughts.

File: project-lifecycle.md [hunk 5]
```diff
@@ -7,24 +7,24 @@ nav_order: 6
...
 collaborative endeavor to deliver a work item.
 There may be some projects that are intended to produce a document, such
 as a requirements or use cases document, a whitepaper, or analysis.
-Others may be to develop a new capability, or refactor (or remove) an
+Other projects develop a new capability, refactor, or remove an
 existing capability for the Hyperledger technology releases. Such
-projects may take the form of a new component (e.g. a new repository) or
-may propose additions, deletions or changes to an existing
-repository(s).
+projects may take the form of a new component (for example, a new repository) or
```
Comment (tkuhrt) [inline project-lifecycle.md]: I believe `e.g.,` is okay

File: project-lifecycle.md [hunk 6]
```diff
@@ -43,40 +43,40 @@ and may go through several iterations.
 
 # Proposal
 
-Project Proposals shall be submitted to the [TSC](https://www.hyperledger.org/about/leadership)
+Project Proposals are submitted to the [TSC](https://www.hyperledger.org/about/leadership)
```
Comment (tkuhrt) [inline project-lifecycle.md]: The verb here could even be `must be`, as this is the only way in which we accept project proposals today.

File: project-lifecycle.md [hunk 7]
```diff
@@ -43,40 +43,40 @@ and may go through several iterations.
...
+-   Identify initial maintainers
+-   Be vendor neutral
 
 # Incubation
 
 Approved project proposals enter into *Incubation*. For new
-components/modules, a repository will be created under the
+components and modules, a repository is created under the
 [Hyperledger Github org](https://github.com/hyperledger)
 and optionally in [JIRA](https://jira.hyperledger.org), if requested. New
```
Comment (tkuhrt) [inline project-lifecycle.md]:
```suggestion
. New
```

We no longer use Jira. Obviously not a grammar issue, but something that we should fix at some point.

File: project-lifecycle.md (hunk 4 above)
Comment (tkuhrt) [inline project-lifecycle.md]:
```suggestion
features or capabilities must be handled through pull requests labeled
```
For consistency with other changes you are making.

File: project-lifecycle.md [hunk 8]
```diff
@@ -43,40 +43,40 @@ and may go through several iterations.
...
-Once a project qualifies to be declared *Graduated*, the
-*project*\'s maintainers can then vote to request a graduation
+After a project qualifies to be declared *Graduated*, the
+*project* maintainers can then vote to request a graduation
 review by the TSC.
 
 Entering *Incubation* does not guarantee that the project will
-eventually get to *Graduated* state. Projects may never get
+eventually get to the *Graduated* state. Projects may never get
 to *Graduated* state.
```
Comment (tkuhrt) [inline project-lifecycle.md]:
```suggestion
to the *Graduated* state.
```
Should we add `the` here to be consistent with previous change?

File: project-lifecycle.md [hunk 9]
```diff
@@ -92,26 +92,25 @@ are in the *Graduated* phase.
 
 # Dormant
 
-Projects in the *Dormant* state are ones in which the normal functions are
-suspended or slowed down for a period of time. The TSC will make the decision
-as to whether a project will move to or from the *Dormant* state upon request.
-If *Dormant* projects become re-activated, they will re-enter the *Incubation*
+Projects enter the *Dormant* state when the normal functions are
+suspended or slowed down for a period of time. The TSC decides to move project to or from the *Dormant* state upon request.
```
Comment (tkuhrt) [inline project-lifecycle.md]:
```suggestion
suspended or slowed down for a period of time. The TSC decides to move a project to or from the *Dormant* state upon request.
```

File: project-lifecycle.md [hunk 10]
```diff
@@ -92,26 +92,25 @@ are in the *Graduated* phase.
...
+Anyone may propose that a project be *Deprecated* by submitting a
+rationale and identifying a substitute project or component, if any. The
 maintainers of the project shall vote on such a request and if it
 passes, make that recommendation to the TSC. Members of the community
-that disagree with the request shall make their case before the TSC. The
-TSC shall consider all points of view and render a final decision to
-deprecate or not.
+that disagree with the request can make their case before the TSC. The
+TSC will consider all points of view and render a final decision to
+deprecate or not deprecate.
```
Comment (tkuhrt) [inline project-lifecycle.md]:
```suggestion
whether to deprecate.
```

File: project-lifecycle.md [hunk 11]
```diff
@@ -92,26 +92,25 @@ are in the *Graduated* phase.
...
-rationale and identifying a substitute project/component (if any). The
+Anyone may propose that a project be *Deprecated* by submitting a
+rationale and identifying a substitute project or component, if any. The
 maintainers of the project shall vote on such a request and if it
 passes, make that recommendation to the TSC. Members of the community
-that disagree with the request shall make their case before the TSC. The
-TSC shall consider all points of view and render a final decision to
-deprecate or not.
+that disagree with the request can make their case before the TSC. The
+TSC will consider all points of view and render a final decision to
```
Comment (tkuhrt) [inline project-lifecycle.md]:
```suggestion
TSC will consider all points of view and render a final decision
```
in combination with my suggestion on the next line

File: project-lifecycle.md (hunk 5 above)
Comment (barriebyron) [inline project-lifecycle.md]:
```suggestion
projects may take the form of a new component (e.g., a new repository) or
```

File: project-lifecycle.md (hunk 6 above)
Comment (barriebyron) [inline project-lifecycle.md]:
```suggestion
Project Proposals must be submitted to the [TSC](https://www.hyperledger.org/about/leadership)
```

**Review by barriebyron — COMMENTED** [review]
thanks for the suggestions!

**Comment by barriebyron** [comment]
closing in favor of https://github.com/hyperledger/tsc/pull/29

**Pull request closed without merging**